src/config.c

```diff
@@ -672,6 +672,9 @@ void loadServerConfigFromString(char *config) {
             server.lua_time_limit = strtoll(argv[1],NULL,10);
         } else if (!strcasecmp(argv[0],"lua-replicate-commands") && argc == 2) {
             server.lua_always_replicate_commands = yesnotoi(argv[1]);
```

Comment:
@trevor211 isn't that redundant?

Reply:
Yes @guybe7, the original PR didn't removed the old statement apparently, fixing. Thank you.

Reply:
Should be ok now.

Reply:
Added another commit actually to make it similar to the other code paths.